protocol/contracts/lpp/src/msg.rs

```diff
@@ -36,7 +37,8 @@ where
     Lpns: Group,
 {
     NewLeaseCode {
-        lease_code_id: Uint64,
+        /// This is an internal system API and we use [Code]
```

Comment:
Perhaps this should be a normal comment, rather than a doc comment, as it doesn't explain nor give other information about the field.